packages/rPython/install

```diff
@@ -1,7 +1,13 @@
 #!/bin/bash
-
 set -x
 set -e
 
+OS_DISTRIBUTION=$(lsb_release -cs)
+
 apt-get update -qq
-apt-get install -y python-dev
+
+if [ $OS_DISTRIBUTION == "xenial" ]; then
+  apt-get install -y python-dev
```

Comment:
isn't python3 the default on xenial for the base now?

Reply:
It turns out the tests pass just fine if we don't install python3-dev at all for either... I think our base image is better up to date now.  I'm going to remove this package as well.